runtime/compiler/p/runtime/PicBuilder.spp

```diff
@@ -174,6 +174,28 @@
 
 #endif
 
+#define SaveFPR                                             \
```

Comment:
What about calling it SaveFPRArgs and RestoreFPRArgs ?

Reply:
I agree to make this change. I think it's a bit on the obvious side, since I can't recall any case we would need to preserve any of the other floating point registers.

Reply:
Not sure it will be so obvious half a year from now :)

Reply:
See https://github.com/eclipse/openj9/pull/7290/commits/fb200ba38a963da8d29ed705a00433acbeb5d211